build.sbt

```diff
@@ -123,7 +123,7 @@ lazy val samples = project
     // FIXME include samples again
     // `java-eventsourced-shopping-cart`
     // `java-eventing-shopping-cart`,
-    // `java-customer-registry`
+    `java-customer-registry`
```

Comment:
Wait, do we want to include in sbt build or not?

Reply:
I think we want to for now. We are using this project as the API experimentation project, so good to compile it each time.